docs/5-user-authentication.md

```diff
@@ -170,21 +48,23 @@ Firebase authentication wasn't entirely compatible with cordova. You need to add
 
 Login with Facebook.
 
-1- Install cordova plugin
+1. Install the cordova plugin
```

Comment:
I think we should to this new docs section instead of providing our own instruction: https://firebase.google.com/docs/auth/web/cordova